libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo/gradle/internal/ApolloDownloadSchemaTask.kt

```diff
@@ -71,6 +71,11 @@ abstract class ApolloDownloadSchemaTask : DefaultTask() {
   @get:Option(option = "insecure", description = "if set to true, TLS/SSL certificates will not be checked when downloading")
   abstract val insecure: Property<Boolean>
 
+  @get:Optional
+  @get:Input
+  @get:Option(option = "failSafeIntrospection", description = "if set to true a minimal introspection query is used and no pre-introspection query")
+  abstract val failSafeIntrospection: Property<Boolean>
```

Comment:
What about falling back to minimal introspection query if everything else fails? 
One less parameter to maintain?

Reply:
Good point! Simpler 👍